projects/storefrontstyles/scss/cxbase/blocks/forms.scss

```diff
@@ -1,250 +1,256 @@
 @import '../../theme';
-@import '../../mixins';
 @import '~bootstrap/scss/forms';
 @import '~bootstrap/scss/custom-forms';
 
 .form-control {
-  margin-bottom: spacer(1);
+  margin-bottom: var(--cx-margin, 0.25rem);
   &:focus {
-    border-color: theme-color('secondary');
-    box-shadow: none;
-    -webkit-box-shadow: none;
+    border-color: var(--cx-border-color, var(--cx-g-color-secondary));
+    box-shadow: var(--cx-box-shadow, none);
+    -webkit-box-shadow: var(--cx-webkit-box-shadow, none);
   }
 }
 
 .invalid-feedback {
-  text-align: left;
+  text-align: var(--cx-text-align, left);
   &::before {
-    content: '!';
-    color: theme-color('inverse');
-    text-align: center;
-    border-radius: 50%;
-    display: inline-block;
-    width: 20px;
-    height: 20px;
-    background-color: theme-color('danger');
-    margin-right: 6px;
-    font-weight: $font-weight-bold;
+    content: var(--cx-content, '!');
+    color: var(--cx-color, var(--cx-g-color-inverse));
+    text-align: var(--cx-text-align, center);
+    border-radius: var(--cx-border-radius, 50%);
+    display: var(--cx-display, inline-block);
+    width: var(--cx-width, 20px);
+    height: var(--cx-height, 20px);
+    background-color: var(--cx-background-color, var(--cx-g-color-danger));
+    margin-right: var(--cx-margin, 6px);
+    font-weight: var(--cx-g-font-weight-normal, 700);
   }
   &:focus {
-    box-shadow: none;
-    -webkit-box-shadow: none;
+    box-shadow: var(--cx-box-shadow, none);
+    -webkit-box-shadow: var(--cx-webkit-box-shadow, none);
   }
 }
 
 .is-invalid {
-  color: theme-color('danger');
+  color: var(--cx-color, var(--cx-g-color-danger));
 }
 
 input[type='checkbox'],
 input[type='radio'] {
-  height: 22px;
-  width: 22px;
-  vertical-align: middle;
-  cursor: pointer;
-  line-height: 1.42857;
-  appearance: none;
+  height: var(--cx-height, 22px);
+  width: var(--cx-width, 22px);
+  vertical-align: var(--cx-vertical-align, middle);
+  cursor: var(--cx-cursor, pointer);
+  line-height: var(--cx-line-height, 1.42857);
+  appearance: var(--cx-appearance, none);
   -webkit-appearance: textfield;
   -moz-appearance: textfield;
   font-size: inherit;
-  box-sizing: border-box;
-  outline: none;
-  border-style: solid;
-  border-width: 1px;
-  position: absolute;
-  margin-top: 0.3rem;
-  margin-left: -1.25rem;
+  box-sizing: var(--cx-box-sizing, border-box);
+  outline: var(--cx-outline, none);
+  border-style: var(--cx-border-style, solid);
+  border-width: var(--cx-border-width, 1px);
+  position: var(--cx-position, absolute);
+  margin-top: var(--cx-margin, 0.3rem);
+  margin-left: var(--cx-margin, -1.25rem);
 }
 
 label {
-  display: block;
+  display: var(--cx-display, block);
   .label-content {
-    display: block;
+    display: var(--cx-display, block);
     margin-bottom: $label-margin-bottom;
     &.bold {
-      font-weight: $font-weight-semi;
+      font-weight: var(--cx-g-font-weight-semi, 600);
```

Comment:
This and the other font-weight should be `var(--cx-font-weight, var(--cx-g-font-weight-semi))` & `var(--cx-font-weight, var(--cx-g-font-weight-normal))`.